gen/exhibitor_tls_bootstrap.py

```diff
@@ -0,0 +1,113 @@
+import json
+import shutil
+import subprocess
+import tempfile
+from pathlib import Path
+from typing import Any, Dict, List
+from urllib.parse import urlparse
+
+PACKAGE_NAME = 'dcos-bootstrap-ca'
+BINARY_PATH = '/genconf/bin'
+CA_PATH = '/genconf/ca'
+
+
+def _check(config: Dict[str, Any]) -> List[str]:
+    """
+    Current constraints are:
+        config.yaml:
+            - exhibitor_tls_enabled == True,
+            - bootstrap_url must not be a local path
+                This is necessary to prevent this orchestration
+                from running when gen is not executed on a proper
+                bootstrap node.
+            - master_discovery must be static
+            - DC/OS variant must be enterprise
+    """
+    checks = [
+        (lambda: config.get('exhibitor_tls_enabled', False) == 'true',
+         'Exhibitor security is disabled'),
+        (lambda: config['master_discovery'] == 'static',
+         'Only static master discovery is supported'),
+        (lambda: config['dcos_variant'] == 'enterprise',
+         'Exhibitor security is an enterprise feature'),
+        (lambda: urlparse(config['bootstrap_url']).scheme != 'file',
+         'Blackbox exhibitor security is only supported when using a remote'
```

Comment:
Lets not use word `Blackbox` here because it is not being used anywhere in the docs or the rest of the code.